backup-parameter-store/src/main/scala/com/gu/backupparameterstore/Lambda.scala

```diff
@@ -0,0 +1,45 @@
+package com.gu.backupparameterstore
+
+import com.amazonaws.services.lambda.runtime.Context
+import com.gu.aws.{ ParameterStore, S3 }
+import com.typesafe.scalalogging.LazyLogging
+
+/**
+ * This is compatible with aws' lambda JSON to POJO conversion.
+ * You can test your lambda by sending it the following payload:
+ * {"name": "Bob"}
+ */
+class LambdaInput() {
```

Comment:
Rather that using this for input, you could also use the relevant event (a scheduled event). E.g. see:

https://github.com/aws/aws-lambda-java-libs/tree/master/aws-lambda-java-events

Note, we don't use the event itself so it may not matter, but it might be clearer than this pojo magic.

Reply:
This comes from a g8 template. It is a bit funky